testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```
@@ -657,7 +706,88 @@ impl SignCoordinator {
                 continue;
             }
 
-            // TODO: get messages from signers (#4775)
+            let modified_slots = &event.modified_slots.clone();
+
+            // Update `next_signers_bitvec` with the slots that were modified in the event
+            modified_slots.iter().for_each(|chunk| {
+                if let Ok(slot_id) = chunk.slot_id.try_into() {
+                    match &self.next_signer_bitvec.set(slot_id, true) {
+                        Err(e) => {
+                            warn!("Failed to set bitvec for next signer: {e:?}");
+                        }
+                        _ => (),
+                    };
+                } else {
+                    error!("FATAL: slot_id greater than u16, which should never happen.");
```

This should just return an error.